dev/provider_packages/prepare_provider_packages.py

```diff
@@ -58,7 +58,7 @@
 # In case you have some providers that you want to have different min-airflow version for,
 # Add them as exceptions here. Make sure to remove it once the min-airflow version is bumped
 # to the same version that is required by the exceptional provider
-MIN_AIRFLOW_VERSION_EXCEPTIONS = {"openlineage": "2.7.0"}
+MIN_AIRFLOW_VERSION_EXCEPTIONS = {"openlineage": "2.7.0", "common.io": "2.8.0"}
```

Comment:
This was missed in the merge of the provider code.
I guess we are missing some kind of test for it